### PR TITLE
ref(storybook): Use Hovercard for storybook icons

### PR DESCRIPTION
**Pull request by evanpurkhiser**
It doesn't behave exactly the same, you can't click them now.

I may bring that back. Just wanted to get this PR up to get it off my work machine

### CODE REVIEW

File: docs-ui/stories/assets/icons/details.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+   */
+  const getCodeSample = useCallback(() => {
+    return `<Icon${icon.name} color="gray500" size="${size}"${
+      isCircled ? ' isCircled' : ' '
+    }${isSolid ? ' isSolid' : ' '} />`;
+  }, [icon.name, size, isCircled, isSolid]);
+
+  const [codeSample, setCodeSample] = useState(getCodeSample());
+
+  useEffect(() => void setCodeSample(getCodeSample()), [getCodeSample]);
```
Comment (vuluongj20) [inline docs-ui/stories/assets/icons/details.tsx:38]: we can probably condense all this into a memoized variable, like:
```jsx
const codeSample = useMemo(() => …, [icon.name, size, isCircled, isSolid])
```

Comment (evanpurkhiser) [reply docs-ui/stories/assets/icons/details.tsx:38]: oh yea, we probably can

**Review by vuluongj20 — APPROVED** [review]
Look's good! I'll spin up another PR to fix the hovercard style.

File: docs-ui/stories/assets/icons/details.tsx (hunk 1 above)
Comment (evanpurkhiser) [reply docs-ui/stories/assets/icons/details.tsx:38]: I'll deal with it later tho lol

**Pull request merged**